client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -0,0 +1,393 @@
+  color: ${prop('primaryTextColor')};
+  padding: ${remSize(8)} 0;
+  gap: ${remSize(14)};
+  display: flex;
+  align-items: center;
+  font-size: ${remSize(10)};
+  box-shadow: #00000030 0px 2px 8px 0px;
+  z-index: 10;
+`;
+
+const LogoContainer = styled.div`
+  width: ${remSize(28)};
+  aspect-ratio: 1;
+  margin-left: ${remSize(14)};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  > svg {
+    width: 100%;
+    height: 100%;
+    > path {
+      fill: ${prop('accentColor')};
+      stroke: ${prop('accentColor')};
+    }
+  }
+`;
+
+const Title = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${remSize(2)};
+
+  * {
+    padding: 0;
+    margin: 0;
+  }
+
+  > h5 {
+    font-size: ${remSize(13)};
+    font-weight: normal;
+  }
+`;
+
+const Options = styled.div`
+  margin-left: auto;
+  display: flex;
+  /* transform: translateX(${remSize(12)}); */
+  svg {
+    fill: ${(props) => props.color};
+  }
+
+  /* Drop Down menu */
+  ul.opened {
+    transform: scale(1);
+    opacity: 1;
+  }
+
+  > div {
+    position: relative;
+    ul {
+      ${prop('MobilePanel.default')}
+      position: absolute;
+      overflow: hidden;
+      z-index: 10;
+      right: 10px;
+      transform: translateX(${remSize(6)});
+      width: max-content;
+      transform: scale(0);
+      opacity: 0;
+      transform-origin: top;
+      transition: opacity 100ms, transform 100ms 200ms;
+      display: flex;
+      flex-direction: column;
+      gap: ${remSize(2)};
+      box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
+        rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+      min-width: ${remSize(120)};
+      border-radius: ${remSize(2)};
+
+      /* User */
+      li.user {
+        color: ${prop('colors.processingBlue')};
+        font-size: ${remSize(16)};
+        font-weight: bold;
+        margin: ${remSize(6)} ${remSize(12)};
+      }
+
+      > b {
+        margin: ${remSize(6)} ${remSize(12)};
+        position: relative;
+        width: min-content;
+        &::after {
+          content: '';
+          position: absolute;
+          opacity: calc(0.5);
+          top: 50%;
+          transform: translate(100%, -50%);
+          right: ${remSize(-6)};
+          width: ${remSize(55)};
+          height: 1px;
+          background-color: ${prop('Button.primary.default.foreground')};
+        }
+      }
+
+      > li {
+        display: flex;
+        width: 100%;
+        a,
+        button {
+          width: 100%;
+          padding: ${remSize(8)} ${remSize(15)} ${remSize(8)} ${remSize(10)};
+          font-size: ${remSize(18)};
+          text-align: left;
+          &:hover {
+            background-color: ${prop('Button.primary.hover.background')};
+            color: ${prop('Button.primary.hover.foreground')};
+          }
+        }
+      }
+    }
+  }
+`;
+
+const LanguageSelect = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  height: 100%;
```

Comment:
The positioning is great if the screen is big enough to fit the whole list.  We need an `overflow-y: auto` somewhere if the list doesn't fit (like in portrait mode of iPhone SE, for example).  I'm playing now and it seems like adding `overflow-y: auto;` on this div and `max-height: 100%; overflow-y: auto;` on the inner div will do it?

Reply:
just `overflow-y: auto;` worked

Reply:
For me at least that doesn't work.  The list is still centered so there's a hidden top part of it that is not reachable.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/4719c7c4-54c1-453e-9c7e-2b90185f163a)
I see "Italiano" first but I should see "Deutsch".